packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -29,7 +29,7 @@ import writeWranglerToml from "./helpers/write-wrangler-toml";
 import type { Config } from "../config";
 import type { WorkerMetadata } from "../create-worker-upload-form";
 import type { KVNamespaceInfo } from "../kv";
-import type { CustomDomainChangeset, CustomDomain } from "../publish";
+import type { CustomDomainChangeset, CustomDomain } from "../publish/publish";
```

Comment:
Purely from a naming perspective the `publish/publish` feels iffy. What about renaming `publish/index.ts` to `publish/options.ts`, and `publish/publish.ts` to `publish/index.ts`?

Reply:
I think we'll need to revisit naming and import/export convention once `index.tsx` is fully modular  - I like the way Pages use barrel exports in `packages/wrangler/src/pages/index.tsx`, could be a way forward